pdfgraph.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Takes a folder of .DOT files and produces PDF files with a graph of each one
+# Maybe use pdfunite?
```

Comment:
Looks like you are using pdfunite.

Reply:
Comment now removed - had been left in by accident